src/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -15,11 +15,22 @@ public static string ToServerString(this Cookie cookie)
         {
             if (s_toServerStringFunc == null)
             {
-                s_toServerStringFunc = (Func<Cookie, string>)typeof(Cookie).GetMethod("ToServerString").CreateDelegate(typeof(Func<Cookie, string>));
+                s_toServerStringFunc = (Func<Cookie, string>)typeof(Cookie).GetMethod("ToServerString", BindingFlags.NonPublic | BindingFlags.Instance).CreateDelegate(typeof(Func<Cookie, string>));
```

Comment:
https://github.com/dotnet/corefx/issues/13607 and for the others I'll update the issue